src/components/sections/challenges/Rewards.tsx

```diff
@@ -19,58 +16,46 @@ import HackathonPrize from "./_partials/HackathonPrize";
 
   return (
     <Section title={`${t("communities.overview.reward.title")}`}>
-      <p className="my-5 text-lg">{t("communities.overview.reward.subtitle")}</p>
-      <div className="md:flex md:flex-row items-center flex-col rounded-full max-w-max text-sm mt-6 space-y-8 md:space-x-8 md:space-y-0">
-        <div className="flex items-center gap-1 md:gap-0">
+      <p className="my-5 text-lg">{t("course.challenge.reward.certificate.description")}</p>
+      <div className="text-sm mt-6 flex gap-8 w-full md:w-2/3">
+        <div className="">
           <Certificate size="medium" name={router.query?.slug as string} />
-          <div className="md:pl-2 space-y-2 max-w-max">
-            <div className="flex text-sm text-gray-700">
-              <span className="block font-medium pr-1">
-                {community?.slug === "celo" && "NFT"} {t("communities.overview.challenge.certificate")}
-              </span>
-            </div>
-            <div className="text-gray-400 text-xs font-medium">{t("communities.overview.challenge.subtitle")}</div>
-          </div>
         </div>
-        {rewards?.map((reward, index) => (
-          <div key={`reward=${index}`} className="flex items-center">
-            <Coin size="medium" token={reward?.token} />
-            <div className="text-sm space-y-2 md:pl-2 max-w-max">
-              {challenge?.isHackathon ? (
-                <HackathonPrize reward={reward} description={t("communities.overview.challenge.rewards")} />
-              ) : (
-                <>
-                  <div className="flex gap-1 text-gray-700 font-medium">
-                    <span>{totalReward}</span>
-                    <span>{reward?.token}</span>
-                    <span>{t("communities.overview.challenge.rewards")}</span>
-                  </div>
-                  <div className="text-gray-400 text-xs font-medium leading-3 mt-1 flex">
-                    {challenge?.rewards.map((reward, index) => (
-                      <span key={`reward-${index}`}>
-                        {index > 0 && "\u003B "}
-                        {reward.amount} {reward.token}/{formatToken(reward.type)}
-                      </span>
-                    ))}
-                  </div>
-                </>
-              )}
-            </div>
+        <div className="flex flex-col lg:flex-row justify-between gap-2 items-start w-full">
+          <div className="flex flex-col w-full lg:w-1/2">
+            {challenge?.rewards.map((reward, index, rewardsArray) => (
+              <div
+                key={reward.id}
+                className={`flex items-center gap-1 pb-2 ${index !== (rewardsArray.length !== 1 && rewardsArray.length - 1) ? "border-b border-gray-200" : ""} ${
+                  index !== 0 ? "pt-2" : ""
+                } `}
+              >
+                <Coin size="small" token={reward?.token} />
+                <div className="text-sm">
```

Comment:
This seems to be a duplicate of what is used in the `cards/Challenge`. We should use the same component